target/linux/ramips/patches-5.15/810-uvc-add-iPassion-iP2970-support.patch

```diff
@@ -225,18 +225,16 @@ Signed-off-by: John Crispin <blogic@openwrt.org>
  	for_each_uvc_urb(uvc_urb, stream) {
 --- a/drivers/media/usb/uvc/uvcvideo.h
 +++ b/drivers/media/usb/uvc/uvcvideo.h
-@@ -209,7 +209,9 @@
+@@ -209,6 +209,8 @@
  #define UVC_QUIRK_RESTORE_CTRLS_ON_INIT	0x00000400
  #define UVC_QUIRK_FORCE_Y8		0x00000800
  #define UVC_QUIRK_FORCE_BPP		0x00001000
--
 +#define UVC_QUIRK_MOTION		0x00001000
 +#define UVC_QUIRK_SINGLE_ISO		0x00002000
```

Comment:
Both of these defines are overlapping with previous definitions. 
Please do something like this:
````
#define UVC_QUIRK_MOTION		0x00004000
#define UVC_QUIRK_SINGLE_ISO		0x00008000
````